packages/vats/src/core/utils.js

```diff
@@ -33,6 +33,9 @@ export const agoricNamesReserved = harden({
     Invitation: 'Zoe invitation',
     timer: 'timer service',
   },
+  chain: {
+    agoric: 'Agoric chain info',
+  },
```

Comment:
Is this used? It suggests that `agoricNames` had a `chain` child all along. It might hide the problem that we need to call `E(agoricNames).publishNameHubs(..., ['chain'])` when running the `orchestration-proposal.js` core eval.

Reply:
> It suggests that agoricNames had a chain child all along

I was on the fence about this but the js doesn't say anything about bootstrap,
https://github.com/Agoric/agoric-sdk/blob/96c19f54bd17fd616883713d372a9dc8ca0140f9/packages/vats/src/core/utils.js#L16-L17

If it means only reserved before the chain boots, I can take this out but I'd want to include new documentation and probably move these `core/utils.js` things that are meant only for bootstrap into a module that makes that clear.

Reply:
Note, this is the change that was necessary to make `publishNameHubs` set up an `onUpdate` to publish each new entry to vstorage.

Reply:
Note, this is the change that was necessary to make `publishNameHubs` set up an `onUpdate` to publish each new entry to vstorage.